interfaces/builtin/mpris.go

```diff
@@ -189,18 +187,23 @@ func (iface *MprisInterface) ConnectedPlugSnippet(plug *interfaces.Plug, slot *i
 func (iface *MprisInterface) PermanentSlotSnippet(slot *interfaces.Slot, securitySystem interfaces.SecuritySystem) ([]byte, error) {
 	switch securitySystem {
 	case interfaces.SecurityAppArmor:
-		snippet := mprisPermanentSlotAppArmor
+		name, err := iface.GetName(slot.Attrs)
```

Comment:
Yes, you can use `export_test.go` to create public variants of private functions. Look for this file in various directories for ideas on how to use it.